index.js

```diff
@@ -29,11 +31,17 @@ class Main {
       .catch((err) => { throw new Error(`${INIT_EXPORTER_ERR_MSG}${err.message}`); });
   }
 
+  
   async handleInitPosition() {
     const lastRecoveredPosition = await this.exporter.getLastPosition();
     this.lastProcessedPosition = this.worker.initPosition(lastRecoveredPosition);
     await this.exporter.savePosition(this.lastProcessedPosition);
   }
+  
+  async #initTaskManager(blockNumber) {
+    this.taskManager = new TaskManager(blockNumber, constants.BLOCK_INTERVAL);
+    await this.taskManager.initPQueue(MAX_CONCURRENT_REQUESTS);
+  }
 
   #isWorkerSet() {
```

Comment:
it's quite common for `isSomething` methods to return `bool` as an answer. This one returns void and confuses me a bit